docs/quickstart.rst

```diff
@@ -151,6 +151,67 @@ The usage for Cross Encoder (a.k.a. reranker) models is similar to Sentence Tran
 
 With ``CrossEncoder("cross-encoder/stsb-distilroberta-base")`` we pick which `CrossEncoder model <./cross_encoder/pretrained_models.html>`_ we load. In this example, we load `cross-encoder/stsb-distilroberta-base <https://huggingface.co/cross-encoder/stsb-distilroberta-base>`_, which is a `DistilRoBERTa <https://huggingface.co/distilbert/distilroberta-base>`_ model finetuned on the `STS Benchmark <https://huggingface.co/datasets/sentence-transformers/stsb>`_ dataset.
 
+Sparse Encoder
+-------------
+
+Characteristics of Sparse Encoder models:
+
+1. Calculates **sparse vector representations** where most dimensions are zero
+2. Provides **efficiency benefits** for large-scale retrieval systems due to the sparse nature of embeddings
+3. Often **more interpretable** than dense embeddings, with non-zero dimensions corresponding to specific tokens
+4. **Complementary to dense embeddings**, enabling hybrid search systems that combine the strengths of both approaches
+
+The usage for Sparse Encoder models follows a similar pattern to Sentence Transformers:
+
+.. sidebar:: Documentation
+
+   1. :class:`SparseEncoder <sentence_transformers.SparseEncoder>`
+   2. :meth:`SparseEncoder.encode <sentence_transformers.SparseEncoder.encode>`
+   3. :meth:`SparseEncoder.similarity <sentence_transformers.SparseEncoder.similarity>`
+   4. :meth:`SparseEncoder.get_sparsity_stats <sentence_transformers.SparseEncoder.get_sparsity_stats>`
+
+   **Other useful methods and links:**
+
+   - `SparseEncoder > Usage <./sparse_encoder/usage/usage.html>`_
+   - `SparseEncoder > Pretrained Models <./sparse_encoder/pretrained_models.html>`_
+   - `SparseEncoder > Training Overview <./sparse_encoder/training_overview.html>`_
+   - `SparseEncoder > Loss Overview <./sparse_encoder/loss_overview.html>`_
+   - `SparseEncoder > Search Integration <./sparse_encoder/search_integration.html>`_
+
+::
+
+   from sentence_transformers import SparseEncoder
+
+   # 1. Load a pretrained SparseEncoder model
+   model = SparseEncoder("naver/splade-cocondenser-ensembledistil")
+
+   # The sentences to encode
+   sentences = [
+       "The weather is lovely today.",
+       "It's so sunny outside!",
+       "He drove to the stadium.",
+   ]
+
+   # 2. Calculate sparse embeddings by calling model.encode()
+   embeddings = model.encode(sentences)
+   print(embeddings.shape)
+   # [3, 30522] - sparse representation with vocabulary size dimensions
+
+   # 3. Calculate the embedding similarities (using dot product by default)
+   similarities = model.similarity(embeddings, embeddings)
+   print(similarities)
+   # tensor([[3.5629e+01, 9.1541e+00, 1.1269e-01],
+   #         [9.1541e+00, 2.7478e+01, 1.9061e-02],
+   #         [1.1269e-01, 1.9061e-02, 2.9612e+01]], device='cuda:0')
+
```

Comment:
```suggestion
   print(similarities)
   # tensor([[   35.6293,     9.1541,     0.1127],
   #         [    9.1541,    27.4778,     0.0191],
   #         [    0.1127,     0.0191,    29.6122]], device='cuda:0')
```
I just figured out that I can use:
```python
import torch
torch.set_printoptions(sci_mode=False)  # Disable scientific notation for better readability
```
and then our examples are a bit clearer. We shouldn't have the `torch.set_printoptions` anywhere visible, but the more readable outputs are quite useful.